build.sbt

```diff
@@ -224,7 +224,7 @@ lazy val sharing = (project in file("sharing"))
     libraryDependencies ++= Seq(
       "org.apache.spark" %% "spark-sql" % sparkVersion % "provided",
 
-      "io.delta" %% "delta-sharing-client" % "1.0.3",
+      "io.delta" %% "delta-sharing-client" % "1.0.4",
```

Comment:
whats new in 1.0.4?

Reply:
mainly about proxy support to access shared tables behind a firewall: https://github.com/delta-io/delta-sharing/releases/tag/v1.0.4